services/madoc-ts/src/routes/projects/update-revision-task.ts

```diff
@@ -29,7 +29,7 @@ export const updateRevisionTask: RouteMiddleware<{ taskId: string; task: any }>
     throw new Error('Task could not be updated, no body');
   }
 
-  if (task.type !== 'crowdsourcing-task') {
+  if (task.type !== 'crowdsourcing-task' || 'crowdsourcing-manifest-task') {
```

Comment:
Might need to also add crowdsouricng-canvas-task too.
```ts
if (
  task.type !== 'crowdsourcing-task' &&
  task.type !== 'crowdsourcing-manifest-task' &&
  task.type !== 'crowdsourcing-canvas-task'
) {
```
(also needs to be `&&`)